src/contracts/misc/GhoCcipSteward.sol

```diff
@@ -0,0 +1,151 @@
+
+  /// @inheritdoc IGhoCcipSteward
+  bool public immutable BRIDGE_LIMIT_ENABLED;
+
+  CcipDebounce internal _ccipTimelocks;
+
+  /**
+   * @dev Only methods that are not timelocked can be called if marked by this modifier.
+   */
+  modifier notTimelocked(uint40 timelock) {
+    require(block.timestamp - timelock > MINIMUM_DELAY, 'DEBOUNCE_NOT_RESPECTED');
+    _;
+  }
+
+  /**
+   * @dev Constructor
+   * @param ghoToken The address of the GhoToken
+   * @param ghoTokenPool The address of the Gho CCIP Token Pool
+   * @param riskCouncil The address of the risk council
+   * @param bridgeLimitEnabled Whether the bridge limit feature is supported in the GhoTokenPool
+   */
+  constructor(
+    address ghoToken,
+    address ghoTokenPool,
+    address riskCouncil,
+    bool bridgeLimitEnabled
+  ) RiskCouncilControlled(riskCouncil) {
+    require(ghoToken != address(0), 'INVALID_GHO_TOKEN');
+    require(ghoTokenPool != address(0), 'INVALID_GHO_TOKEN_POOL');
+
+    GHO_TOKEN = ghoToken;
+    GHO_TOKEN_POOL = ghoTokenPool;
+    BRIDGE_LIMIT_ENABLED = bridgeLimitEnabled;
+  }
+
+  /// @inheritdoc IGhoCcipSteward
+  function updateBridgeLimit(
+    uint256 newBridgeLimit
+  ) external onlyRiskCouncil notTimelocked(_ccipTimelocks.bridgeLimitLastUpdate) {
+    require(BRIDGE_LIMIT_ENABLED, 'BRIDGE_LIMIT_DISABLED');
+
+    uint256 currentBridgeLimit = IUpgradeableLockReleaseTokenPool(GHO_TOKEN_POOL).getBridgeLimit();
+    require(newBridgeLimit != currentBridgeLimit, 'NO_CHANGE_IN_BRIDGE_LIMIT');
+    require(
+      _isDifferenceLowerThanMax(currentBridgeLimit, newBridgeLimit, currentBridgeLimit),
+      'INVALID_BRIDGE_LIMIT_UPDATE'
+    );
+
+    _ccipTimelocks.bridgeLimitLastUpdate = uint40(block.timestamp);
+
+    IUpgradeableLockReleaseTokenPool(GHO_TOKEN_POOL).setBridgeLimit(newBridgeLimit);
+  }
+
+  /// @inheritdoc IGhoCcipSteward
+  function updateRateLimit(
+    uint64 remoteChainSelector,
+    bool outboundEnabled,
+    uint128 outboundCapacity,
+    uint128 outboundRate,
+    bool inboundEnabled,
+    uint128 inboundCapacity,
+    uint128 inboundRate
+  ) external onlyRiskCouncil notTimelocked(_ccipTimelocks.rateLimitLastUpdate) {
+    RateLimiter.TokenBucket memory outboundConfig = IUpgradeableLockReleaseTokenPool(GHO_TOKEN_POOL)
+      .getCurrentOutboundRateLimiterState(remoteChainSelector);
+    RateLimiter.TokenBucket memory inboundConfig = IUpgradeableLockReleaseTokenPool(GHO_TOKEN_POOL)
+      .getCurrentInboundRateLimiterState(remoteChainSelector);
+
+    require(
+      outboundEnabled != outboundConfig.isEnabled ||
+        outboundCapacity != outboundConfig.capacity ||
+        outboundRate != outboundConfig.rate ||
+        inboundEnabled != inboundConfig.isEnabled ||
+        inboundCapacity != inboundConfig.capacity ||
+        inboundRate != inboundConfig.rate,
+      'NO_CHANGE_IN_RATE_LIMIT'
+    );
+
+    require(
+      _isDifferenceLowerThanMax(outboundConfig.capacity, outboundCapacity, outboundConfig.capacity),
```

Comment:
A 100% change of 0 is 0. Given that current rate limit is not set (=0), steward won't be able to update it.

Reply:
One potential strategy is to have a minimum amount that we can change, even if it's 0. Or maybe we can discuss the possibility of using config not based on the current value, and allowing linear changes, like we have for borrow rate

Reply:
It's fine as it is, assuming this is something that must go through AIP process.